I18n.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+using System.Resources;
+
+namespace OFGB
+{
+    public class I18n : INotifyPropertyChanged
+    {
+        private readonly ResourceManager manager;
+
+        private static readonly Lazy<I18n> lazy = new(() => new I18n());
+        public static I18n Instance => lazy.Value;
+        public event PropertyChangedEventHandler? PropertyChanged;
+
+        public I18n()
+        {
+            manager = new ResourceManager("OFGB.Resources.i18n", typeof(I18n).Assembly);
+        }
+
+        public string this[string name]
+        {
+            get
+            {
+                ArgumentNullException.ThrowIfNull(name);
+                return manager.GetString(name) ?? string.Empty;
+            }
+        }
+
+        public void ApplyLanguage(CultureInfo cultureInfo)
+        {
+            CultureInfo.CurrentCulture = cultureInfo;
+            CultureInfo.CurrentUICulture = cultureInfo;
+            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs("item[]"));
```

Comment:
In relation to the comment for passing CultureInfo.CurrentCulture, we could have a check if the passed-in culture is supported and if not just default to English?

Reply:
You are right and thank you for your review. But I would like the suggestion from the repo owner. If she thinks my solution for i18n is OK, I will continue. Otherwise, my work is meaningless here.

Reply:
> In relation to the comment for passing CultureInfo.CurrentCulture, we could have a check if the passed-in culture is supported and if not just default to English?

Thats what I was thinking, if there is an easy way to get their current lang then we can use that as the default.

Possibly helpful discussion https://stackoverflow.com/questions/5710127/get-operating-system-language-in-c-sharp

Reply:
I also plan to look at accessibility stuff for vision impaired folks to make sure its all good.

Reply:
> Thats what I was thinking, if there is an easy way to get their current lang then we can use that as the default.
> 

CultureInfo.CurrentUICulture should be sufficient here.